.github/workflows/backup-db.yml

```diff
@@ -0,0 +1,40 @@
+name: Supa-backup
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
```

Comment:
I think it's enough to keep the `workflow_dispatch` and `schedule` handlers. Performing a backup on push and PR changes seems to be an overkill.